packages/api-plugin-fulfillment/src/preStartup.js

```diff
@@ -0,0 +1,21 @@
+import Logger from "@reactioncommerce/logger";
+import ReactionError from "@reactioncommerce/reaction-error";
+import { extendFulfillmentSchemas } from "./simpleSchemas.js";
+
+const logCtx = { name: "fulfillment", file: "preStartup" };
+
+/**
+ * @summary Called before startup to extend schemas
+ * @param {Object} context Startup context
+ * @returns {undefined}
+ */
+export default async function fulfillmentPreStartup(context) {
+  const allFulfillmentTypesArray = context.fulfillment?.registeredFulfillmentTypes;
+
+  if (!allFulfillmentTypesArray || allFulfillmentTypesArray.length === 0) {
```

Comment:
This will never be true because this plugin defined the `undecided` fulfillmentType so no need for the check.

Reply:
Thank you. Removed.
Will be pushing the changes soon